src/Routes.js

```diff
@@ -12,6 +15,10 @@ class Routes extends React.Component {
     return (
       <Router>
         <Switch>
+          <Route exact path="/Allshop" component={Allshop} />
+          <Route exact path="/Gbnow" component={Gbnow} />
+          <Route exact path="/SlideMenu" component={SlideMenu} />
```

Comment:
- 라우트 경로 다시 한번 확인해주세요! 경로는 소문자로 작성해 주시는게 일반적입니다! (띄어쓰기 부분은 `-` 로 처리합니다.)
```suggestion
          <Route exact path="/allshop" component={Allshop} />
          <Route exact path="/gbnow" component={Gbnow} />
          <Route exact path="/slide-menu" component={SlideMenu} />
```